src/VisualStudio/Core/Test/Snippets/SnippetTestState.vb

```diff
@@ -148,6 +138,58 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Snippets
             MyBase.SendTypeChars(typeChars, AddressOf handler.ExecuteCommand)
         End Sub
 
+        Public Async Function AssertNoCompletionSession() As Task
```

Comment:
Why are we copy/pasting this from the other file too?

Reply:
Just to pass tests for now. Definitely will merge them together.